scipy/stats/_stats_py.py

```diff
@@ -5283,6 +5316,12 @@ def kendalltau(x, y, *, nan_policy='propagate',
            The p-value for a hypothesis test whose null hypothesis is
            an absence of association, tau = 0.
 
+    Raises
+    ------
+    ValueError
+        If `nan_policy` is `omit` and `variant` is not `b`.
+        If `method` is `exact` and there are ties in `x` and `y`.
```

Comment:
See [recommendations](https://numpydoc.readthedocs.io/en/latest/format.html#common-rest-concepts) regarding backticks. Single backticks are correct for parameters like `nan_policy` and `method`, but you need double backticks (and quotes, technically) for literals like `'omit'` and `'b'`.